tasks/main.yml

```
@@ -119,10 +123,11 @@
     - perun_config
 
 - name: "perun ldapc setup"
-  import_tasks: perun_ldapc.yml
+  include_tasks: perun_ldapc.yml
```

tags need to be applied for included tasks:
```suggestion
  include_tasks: perun_ldapc.yml
    apply:
      tags:
        - perun_ldapc
        - perun_config
```